dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/stack/transport/netty/SipMessageBufferStreamDecoder.java

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.sip.stack.transport.netty;
+
+import java.util.List;
+
+import com.ibm.ws.sip.stack.transaction.transport.connections.SipMessageByteBuffer;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.handler.codec.ByteToMessageDecoder;
+
+public class SipMessageBufferStreamDecoder extends ByteToMessageDecoder {
+
+	@Override
+	protected void decode(ChannelHandlerContext ctx, ByteBuf in, List<Object> out) throws Exception {
+		// System.out.println("decode. " + in.toString(Charset.forName("utf-8")));
```

Comment:
If only a prototype debug sys out, we should clean and remove these (two in this method)

Reply:
removed